cherab/core/model/lineshape.pyx

```diff
@@ -352,7 +386,8 @@ cdef class StarkBroadenedLine(LineShapeModel):
         Line(tritium, 0, (9, 3)): (5.588e-15, 0.7165, 0.033)
     }
 
-    def __init__(self, Line line, double wavelength, Species target_species, Plasma plasma, dict stark_model_coefficients=None):
+    def __init__(self, Line line, double wavelength, Species target_species, Plasma plasma, Integrator1D integrator=None,
+                 dict stark_model_coefficients=None):
```

Comment:
Please consider putting integrator parameter at the end for backwards compatibility

Reply:
Done.